src/services/meteorsData.js

```diff
@@ -0,0 +1,38 @@
+const axios = require('axios');
+const moment = require('moment/moment');
+require('dotenv').config();
+
+const { API_BASE_URL } = process.env;
+const { API_KEY } = process.env;
+const START_DATE = moment().format('YYYY-MM-DD');
+const END_DATE = moment().add(7, 'days').format('YYYY-MM-DD');
+const API_SERVICE_URL = `${API_BASE_URL}?start_date=${START_DATE}&end_date=${END_DATE}&api_key=${API_KEY}`;
```

Comment:
instead of using string format to create URL with query params, you can try 
```
await axios.get(API_BASE_URL, {
        params: {
            start_date: START_DATE,
            end_date: END_DATE,
            api_key: API_KEY
        }
```

Reply:
Good idea, thanks!